app/controllers/placements/support/providers_controller.rb

```diff
@@ -1,31 +1,32 @@
 class Placements::Support::ProvidersController < Placements::Support::ApplicationController
   def new
-    @provider = Provider.new
+    @provider_form = ProviderForm.new
```

Comment:
Same same

```suggestion
    @provider_onboarding_form = ProviderOnboardingForm.new
```

Reply:
resolved in [8a29568](https://github.com/DFE-Digital/itt-mentor-services/pull/105/commits/8a2956808a0e940e50960406e039c4df9a898787)